sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -88,6 +88,11 @@
      */
     public static final String PROPERTY_AZURE_IDENTITY_DISABLE_CP1 = "AZURE_IDENTITY_DISABLE_CP1";
 
+    /**
+     * Name of Azure AAD regional authority.
+     */
+    public static final String PROPERTY_AZURE_REGIONAL_AUTHORITY_NAME = "AZURE_REGIONAL_AUTHORITY_NAME";
```

Comment:
@jianghaolu This property is not included in `DEFAULT_CONFIGURATIONS` array. All other properties are included. Is there a reason we don't want this to be included?